webapp/src/components/systemConsole/surveyResults/surveyResults.tsx

```diff
@@ -119,6 +155,18 @@ function SurveyResults() {
                 </div>
             </div>
 
+            {
+                showConfirmationModal &&
+                <ConfirmationModal
+                    id='endSurveyConfirmationModal'
+                    title='Are you sure you want to end this survey?'
+                    bodyMessage='Reprehenderit aute laborum anim dolore magna mollit incididunt officia qui labore. Esse culpa ipsum ut esse fugiat do tempor duis mollit. Labore nisi nisi ut aliquip enim aute ex. Esse veniam proident id dolor laborum occaecat voluptate esse laborum cupidatat aliquip ut laborum id. Laborum nostrud officia excepteur. Consectetur ad nisi ullamco est cillum proident Lorem. Aliqua eu est magna deserunt est elit elit do.'
```

Comment:
Do we want this Lorem ipsum to get to main?

Reply:
We'll update it with other text review. Its fine.